doc/api-extensions.md

```diff
@@ -2324,3 +2324,8 @@ This introduces a new `io.bus` property to disk devices which can be used to ove
 
 ## `storage_cephfs_create_missing`
 This introduces the configuration keys `cephfs.create_missing`, `cephfs.osd_pg_num`, `cephfs.meta_pool` and `cephfs.osd_pool` to be used when adding a `cephfs` storage pool to instruct LXD to create the necessary entities for the storage pool, if they do not exist.
+
+## `instance_move_config`
```

Comment:
Please can you include this as part of the earlier commit and use the prefix `api: ...`